### ISSUE TITLE
Random id for resources in dev mode

### ISSUE THREAD

**Issue by akloeckner** [resoai/TileBoard#451]
I noticed that (running `npm run dev`) my tileboard would not update on `F5`, if it is served over my usual channel, i.e. the HA server. I suggest to add an `?id=Date.now()` to the ressources in dev mode, as a convenience. (Although it is obviously possible to use the dev server on port `8080` and everything works fine.)

**Comment by rchl** [comment]
So you are running a development server on PI (or wherever you are running HA) but serving the files through HA's server?

Maybe you want to run `npm run build:watch` instead then? This creates production bundles and those have a hash so any change will force the browser to fetch new files.
(The downside is that those builds are slower but you can customize it to make it faster)

Or alternatively, develop with browser developer tools opened and "Disable cache" option enabled?
![Screenshot 2020-10-01 at 19 27 15](https://user-images.githubusercontent.com/153197/94842815-1e5def00-041c-11eb-8597-bf245dec18cb.png)

**Comment by akloeckner** [comment]
Yes, that's what I do (did at least).

Will try your suggestions!

**Comment by akloeckner** [comment]
Maybe you also have a hint on how to develop / debug `less` files now? I find it hard to follow the changes they generate in the css, if the css cannot be diff'ed.

**Comment by rchl** [comment]
From the reviewer's perspective, I don't really miss seeing the CSS changes. Seeing the LESS changes is IMO enough to understand and review the change. It's even easier than looking at generated CSS.

From the development perspective, I also don't miss anything. I can see generated CSS and also it links to respective places in LESS files thanks to source maps.

So I'm not sure I understand the issue.

**Comment by akloeckner** [comment]
I'm doing some refactoring on the less code as well. And I'd like to make sure, the result is identical. Source maps are the cue, I was maybe missing. 

Also good to know, you'll do fine reviewing without the diff...

**Comment by rchl** [comment]
I think there are enough options to handle this case already:
 - use production bundles so that resources are hashed
 - use dev server to avoid caching
 - develop with developer tools open with `Disable cache` option enabled.
